src/ClusterBootstrap/scripts/install-blobfuse.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+sudo rm -f packages-microsoft-prod.deb
+wget https://packages.microsoft.com/config/ubuntu/16.04/packages-microsoft-prod.deb
```

Comment:
Just want to confirm if 16.04 is correct

Reply:
Yes, this is correct, they did not provide 18.04 but I have tested it, it works as expected.